sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -546,6 +546,41 @@ val GPU_COREDUMP_PIPE_PATTERN = conf("spark.rapids.gpu.coreDump.pipePattern")
     .bytesConf(ByteUnit.BYTE)
     .createWithDefault(-1)
 
+  val PARTIAL_FILE_BUFFER_INITIAL_SIZE = 
+    conf("spark.rapids.memory.host.partialFileBufferInitialSize")
+    .doc("The initial size in bytes for a host memory buffer used by " +
```

Comment:
I would like to understand where you got the default values, and it would be nice to call out how this and the other configs interact with each other. It is also not clear how these relate to shuffle. I get that these are internal configs, but just from the description I don't know anything about what they do. I don't knwo what a SpillablePartialFileHandle is.